specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -381,6 +381,92 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HanaOnAzure/hanaInstances/{hanaInstanceName}/start": {
```

Comment:
please model "default" response for all your APIs. Sample here - https://github.com/Azure/azure-rest-api-specs/blob/master/specification/managedservices/resource-manager/Microsoft.ManagedServices/stable/2019-06-01/managedservices.json#L62